src/CMakeLists.txt

```
@@ -0,0 +1,79 @@
+# set minimum cmake version
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+
+# Uncomment for VERBOSE Makefiles. Useful for debugging.
+#set(CMAKE_VERBOSE_MAKEFILE ON)
+
+# Uncomment for Debug Build
+#set(CMAKE_BUILD_TYPE Debug)
+set(CMAKE_BUILD_TYPE Release)
```

I overlooked this in the IMPACT-T PR: This should read preferrably
```suggestion
if(NOT DEFINED CMAKE_BUILD_TYPE)
    set(CMAKE_BUILD_TYPE Release)
endif()
```
otherwise users cannot change this anymore from the command line.